test/framework/workload.go

```diff
@@ -21,3 +32,71 @@ func (w *WorkloadCluster) DeleteCluster(opts ...CommandOpt) {
 	opts = append(opts, withKubeconfig(w.ManagementClusterKubeconfigFile()))
 	w.deleteCluster(opts...)
 }
+
+// CreateClusterWithKubectl creates a cluster by applying a workload cluster config using the kubectl client.
+func (w *WorkloadCluster) CreateClusterWithKubectl() {
+	ctx := context.Background()
+	w.T.Logf("Applying workload cluster %s spec for creation located at %s", w.ClusterName, w.ClusterConfigLocation)
+	w.applyClusterManifest(ctx)
+	w.waitForKubeconfig(ctx)
+	time.Sleep(2 * time.Minute)
```

Comment:
Is this number (2 mins) derived from the average amount of time it takes for everything to be ready?

Reply:
This sleep ideally shouldn't be here, but the test was throwing a `connect: network is unreachable` when trying to use it. Working on fixing this by implementing retries where necessary.